tests/saitests/py3/sai_qos_tests.py

```diff
@@ -4190,13 +4190,19 @@ def runTest(self):
 
         cell_occupancy=(packet_length + cell_size - 1) // cell_size
 
-        pkt=simple_tcp_packet(pktlen=packet_length,
-                                eth_dst=router_mac if router_mac != '' else dst_port_mac,
-                                eth_src=src_port_mac,
-                                ip_src=src_port_ip,
-                                ip_dst=dst_port_ip,
-                                ip_tos=tos,
-                                ip_ttl=ttl)
+        pkt = get_multiple_flows_udp(
+                self,
+                router_mac if router_mac != '' else dst_port_mac,
```

Comment:
do we need similar change for other test case also ?

Reply:
Yes, they can also use this change.

Reply:
@neethajohn i am surprise why we don;t see this issue more oftern ? When destination port is lag we are not using same packet that calculate lag member port and the one we are using from test case perspective itself.